extensions/aws/s3/provision/src/main/java/org/eclipse/dataspaceconnector/provision/aws/s3/S3BucketProvisioner.java

```diff
@@ -45,12 +106,9 @@ public S3BucketProvisioner(ClientProvider clientProvider, int sessionDuration, M
         this.clientProvider = clientProvider;
         this.sessionDuration = sessionDuration;
         this.monitor = monitor;
-        this.retryPolicy = retryPolicy;
-    }
-
-    @Override
-    public void initialize(ProvisionContext context) {
-        this.context = context;
+        this.retryPolicy = retryPolicy.copy()
+                .withMaxRetries(10)
```

Comment:
Can you make this be configurable with a default that is represented as a constant?